api/core/tools/provider/builtin/searxng/searxng.yaml

```diff
@@ -20,4 +20,5 @@ credentials_for_provider:
       zh_Hans: 请输入您的 SearXNG base URL
     placeholder:
       en_US: Please input your SearXNG base URL
-      zh_Hans: 请输入您的 SearXNG base URL
\ No newline at end of file
+      zh_Hans: 请输入您的 SearXNG base URL
+    url: https://docs.dify.ai/tutorials/tool-configuration/searxng
```

Comment:
Please update this file as well, otherwise it will not show on the docs site.
https://github.com/langgenius/dify-docs/blob/201d16be461c81cc42036ac4be64c118b5f86cd1/zh_CN/SUMMARY.md